tests/testConfig.sublime-workspace

```diff
@@ -0,0 +1,449 @@
+{
```

Comment:
This doesn't belong, I think.

I had been meaning to ask what editor you use, but don't need to anymore...